packages/web-react/docs/stories/examples/FooterContent.stories.tsx

```diff
@@ -0,0 +1,371 @@
+import React from 'react';
+import { ButtonLink } from '../../../src/components/ButtonLink';
+import { Container } from '../../../src/components/Container';
+import { Divider } from '../../../src/components/Divider';
+import { Flex } from '../../../src/components/Flex';
+import { Footer } from '../../../src/components/Footer';
+import { Grid } from '../../../src/components/Grid';
+import { Icon } from '../../../src/components/Icon';
+import { Link } from '../../../src/components/Link';
+import { Select } from '../../../src/components/Select';
+import { Stack } from '../../../src/components/Stack';
+import { Text } from '../../../src/components/Text';
+import { UNSTABLE_ProductLogo } from '../../../src/components/UNSTABLE_ProductLogo';
+import { defaultSvgLogo } from '../../../src/components/UNSTABLE_ProductLogo/demo/ProductLogoDefault';
+import { VisuallyHidden } from '../../../src/components/VisuallyHidden';
+import { GridColumns } from '../../../src/types';
+
+type FooterCompositionsProps = {
+  headingsWithLink: boolean;
+  nestedLinkBlocks: boolean;
+  numberOfLinkColumns: GridColumns;
+  showDividers: boolean;
+  showLanguageSwitch: boolean;
+  showSecondaryLinks: boolean;
+  showSocialMediaButtons: boolean;
+};
+
+export default {
+  title: 'Examples/Compositions',
+  argTypes: {
+    headingsWithLink: {
+      control: 'boolean',
+      description: 'Headings displayed as links',
+      table: {
+        defaultValue: { summary: false },
+      },
+    },
+    nestedLinkBlocks: {
+      control: 'boolean',
+      description: 'Nested menu in **1st column**',
+      table: {
+        defaultValue: { summary: true },
+      },
+    },
+    numberOfLinkColumns: {
+      control: 'select',
+      options: [0, 1, 2, 3, 4],
```

Comment:
I'd like to see up to 8 columns here. They would display like this:

- 5: 2 on tablet, 5 on desktop,
- 6–8: 2 on tablet, 4 on desktop.